transport/src/main/java/io/netty/channel/AbstractChannel.java

```diff
@@ -810,6 +822,10 @@ public final void flush() {
                 return;
             }
 
+            flush(outboundBuffer);
```

Comment:
@nmittler that will be a similar model of what @ejona86 is suggesting with `ELIDE_FLUSHING. I believe that these are two different approaches with different benefits.